myapp/spec/system/task_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'rails_helper'
+
+RSpec.describe 'Task', type: :system do
+  before do
+    @user = create(:user)
+    @task1 = create(:task, title: 'Test Task 1', status: 0)
+    @task2 = create(:task, title: 'Test Task 2', status: 1)
+    @task3 = create(:task, title: 'Test Task 3', status: 2)
```

Comment:
Ah! Login step is for the later step.
I think i should remove the @user from here since there are no use of it for now.
And for creating task object dynamic. i think it makes sense to be inside `it`. 
will change accordingly

Reply:
modified